gaia-text-input.js

```diff
@@ -148,6 +149,15 @@ module.exports = component.register('gaia-text-input', {
       }
     },
 
+    inputmode: {
+      get: function() { return this.els.input.getAttribute('x-inputmode'); },
+      set: function(value) {
+        if (!value) { return; }
```

Comment:
OK happy not to perform check then, but let's make sure to remove it when falsy :)